packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -146,6 +146,58 @@ class HotRunner extends ResidentRunner {
     throw 'Failed to compile $expression';
   }
 
+  @override
+  Future<void> reloadMethod({String libraryId, String classId}) async {
+    final Stopwatch stopwatch = Stopwatch()..start();
+    final UpdateFSReport results = UpdateFSReport(success: true);
+    final List<Uri> invalidated =  <Uri>[Uri.parse(libraryId)];
+    for (FlutterDevice device in flutterDevices) {
+      results.incorporateResults(await device.updateDevFS(
```

Comment:
Could these be done in parallel via `Future.wait`?

Reply:
It could, but this code is optimized for readability since we likely won't support -d all with this mode.